content/blogs/2024-04-20-top-10-cool-features-I-love-about-kestra.md

```diff
@@ -0,0 +1,150 @@
+
+## 10. Output Preview
+
+Multiple orchestration tasks generate data, either by fetching it from external systems, or by performing actions on top of the existing data. These resultant data sets are stored internally by the orchestration engines. As part of the orchestration steps, these data sets are uploaded to file systems like S3 and Blob Storage, and used for further processing. But many times, you want to check the contents of the data sets being produced to ensure you are getting the desired results. The only choice you are left with is to download the data from the tool's internal storage onto your local machine. Soon, your local machine is cluttered with these regularly downloaded files. 
+
+Kestra provides an extremely smart and easy to use feature of previewing these data sets. You can go to the Outputs tab, and the data sets that are downloaded to internal storage are available with Download and Preview option. The [Preview option](https://kestra.io/docs/workflow-components/outputs#outputs-preview) is one of my favorites giving me quick access to look at the output file contents. This is how the Preview of the data set looks like:
+
+![output_preview](/blogs/2024-04-20-top-10-cool-features-I-love-about-kestra/output_preview.png)
+
+## 9. Editor
+
+Kestra comes with an in-built [VSCode editor](https://kestra.io/docs/getting-started/ui#editor). For any coding that you need to do, you need not go anywhere outside this tool. You have the well-equipped VS Code editor right there within Kestra itself. It also comes with the Kestra extension installed out of the box, and is flexible to also install any other extensions of your choice. This comes in very handy when you want to write scripts to be used within your orchestration flow.
+
+![vscode_editor](/blogs/2024-04-20-top-10-cool-features-I-love-about-kestra/vscode_editor.png)
+
+## 8. Auto-suggestion for task type
+
+When you become a bit used-to to Kestra, you create the new flow, and start typing out your tasks. While you know that you want to write the task pertaining to certain third party plugin, it can become difficult to memorize the actual task type. But going to the plugin documentation every time just to get the task type is pretty time consuming and exhausting. Kestra has an ultimate solution to this problem. After typing out `task: `, you can start typing out any part of the type content like the plugin name, and you would get auto-suggestions containing what you have typed. This has saved multiple minutes of my time on a daily basis.
```

Comment:
```suggestion
When you become used to Kestra, you create the new flow and start typing out your tasks. While you know that you want to write the task pertaining to certain third-party plugins, it can become difficult to memorize the actual task type. But going to the plugin documentation every time just to get the task type is pretty time-consuming and exhausting. 

Kestra has an elegant solution to this problem. After typing out `task: `, you can start typing out any part of the type content, like the plugin name, and you will get auto-suggestions containing what you have typed. This has saved multiple minutes of my time on a daily basis.
```